src/storage/mod.rs

```diff
@@ -262,6 +263,23 @@ impl Command {
 
 use util::transport::SendCh;
 
+#[derive(Default)]
+pub struct OptionalArgs {
```

Comment:
`OptionalArgs` doesn't seem to be a good name, maybe call it `Options` or `RwOption`?